server/src/tenant/entity-resolver/entity-resolver.service.ts

```diff
@@ -6,12 +6,20 @@ import {
 
 import { GraphQLResolveInfo } from 'graphql';
 import graphqlFields from 'graphql-fields';
+import { v4 as uuidv4 } from 'uuid';
 
 import { DataSourceService } from 'src/metadata/data-source/data-source.service';
 import { EnvironmentService } from 'src/integrations/environment/environment.service';
+import { pascalCase } from 'src/utils/pascal-case';
 
 import { convertFieldsToGraphQL } from './entity-resolver.util';
 
+function stringify(obj: any) {
```

Comment:
`stringifyWithoutQuote`?

Reply:
@Weiko Yes, I'll definitely change this to something better